std/format/internal/write.d

```diff
@@ -569,8 +592,8 @@ if (is(FloatingPointTypeOf!T) && !is(T == enum) && !hasToString!(T, Char))
     {
         static if (is(T == float) || is(T == double) || (is(T == real) && T.mant_dig == double.mant_dig))
         {
-            import std.math;
-            import std.format.internal.floats : RoundingMode, printFloat;
+            import std.format.internal.floats : printFloat, RoundingMode;
+            import std.math; // cannot be selectiv, because FloatingPointControl might not be defined
```

Comment:
```suggestion
            import std.math; // cannot be selective, because FloatingPointControl might not be defined
```
This goes for the rest of the comments also.